design_system/pubspec.yaml

```
@@ -24,6 +24,12 @@ dev_dependencies:
 flutter:
 
   # To add assets to your package, add an assets section, like this:
+  assets:
+    - assets/icons/fi-rs-checked.png
+    - assets/images/coo_brand_horizontal.png
+    - assets/images/coo_no_coop.png
+    - assets/images/coo_sucess.png
+    - assets/images/coo_brand.png
```

```suggestion
  assets:
    - assets/icons/
    - assets/images/
```
Vai evitar o retrabalho de adicionar o caminho de novos assets